gobblin-yarn/src/main/java/org/apache/gobblin/yarn/GobblinApplicationMaster.java

```diff
@@ -227,6 +227,8 @@ public static void main(String[] args) throws Exception {
         printUsage(options);
         System.exit(1);
       }
+      Config config = ConfigFactory.load();
```

Comment:
Can we replace the ConfigFactory.load() in line 240 with the config variable created here?